IdentityCore/src/controllers/broker/MSIDBrokerInteractiveController.m

```diff
@@ -126,6 +126,36 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
      }];
 }
 
+- (BOOL)canPerformRequest
```

Comment:
I'd prefer to check for ability to use a particular controller before initializing a controller

Reply:
done